server.js

```diff
@@ -44,36 +54,170 @@ app.post('/api/events', (req, res) => {
+  const title = req.body.title;
+  const location = req.body.location;
+  const startDate = req.body.startDate;
+  const endDate = req.body.endDate;
+  const description = req.body.description;
+  const queryStringEvent = "INSERT INTO Event (Title, StartDate, EndDate, Description, LocationAddress, OrganizerUserID) VALUES (?, ?, ?, ?, ?, ?)";
+  const queryStringOrganizer = "INSERT INTO Organizer (OrganizerUserID, OrganizerContactInfo) VALUES (?, ?)";
+  const queryStringEventType = "INSERT INTO EventType (TypeName, AgeLimit) VALUES (?, ?)";
+
+  const typeNames = ["Family Event", "Overnight Party", "Job Fair", "Auto Show", "National Day Fireworks", "Outdoor Movie", "Debate"];
+  const randomNumber = Math.floor(Math.random() * typeNames.length);
+
+  const ageLimits = [100, 18, 21, 19, 65];
+  const randomAge = Math.floor(Math.random() * ageLimits.length);
+
+  getConnection().query(queryStringEvent, [title, startDate, endDate, description, location, organizerID], (err, results, fields) => {
+    if (err) {
+      console.log("Failed to create new event: " + err);
+      res.sendStatus(500);
+      return;
+    }
+    console.log("Inserted a new event");
+  });
+
+  getConnection().query(queryStringOrganizer, [organizerID, organizerContactInfo], (err, results, fields) => {
+    if (err) {
+      console.log("Failed to create new entry in Organizer: " + err);
+      res.sendStatus(500);
+      return;
+    }
+    console.log("New entry created in Organizer");
+  });
+
+  getConnection().query(queryStringEventType, [typeNames[randomNumber], ageLimits[randomAge]], (err, results, fields) => {
+    if (err) {
+      console.log("Failed to create new entry in EventType");
+      res.sendStatus(500);
+      return;
+    }
+    console.log("New entry created in EventType");
+  })
+
+  res.sendStatus(200);
+  return;
 })
 
 // Delete the event with given eventID
 app.delete('/api/event/:eventID', (req, res) => {
-  console.log(req.params.eventID);
-  res.send();
+  console.log("Deleting event with ID = " + req.params.eventID);
+  const eventId = req.params.eventID;
+  const queryString = "DELETE FROM Event WHERE EventID = ?";
+
+  getConnection().query(queryString, [eventId], (err, results, fields) => {
+    if (err) {
+      console.log("Failed to delete event " + eventId);
+      res.sendStatus(500);
+      return;
+    }
+    console.log("Event deleted!");
+    res.sendStatus(200);
+    return;
+  });
+
 })
 
 // User with userID joins event with eventID. Create such
 // entry in Participate table
 app.put('/api/event/:eventID/user/:userID', (req, res) => {
-  console.log(req.params.eventID);
-  console.log(req.params.userID);
-  res.send();
+  console.log("Updating attendee list for event: " + req.params.eventID);
+  const userId = req.params.userID;
+  const eventId = req.params.eventID;
+  const statusArray = ['Interested', 'Registered', 'Paid'];
+  const randomNumber = Math.floor(Math.random() * statusArray.length);
```

Comment:
To make it easy, let's just set all status to 'Registered'. We are not using it in our frontend anyways.